src/hooks/index.js

```diff
@@ -165,7 +208,10 @@ exports.removeResponse = (excludeOptions) => (context) => {
 	if (excludeOptions === undefined) {
 		excludeOptions = ['get', 'find'];
 	}
-	if (Array.isArray(excludeOptions) && excludeOptions.includes(context.method)) {
+	if (
+		Array.isArray(excludeOptions) &&
```

Comment:
Codacy found an issue: ['&&' should be placed at the beginning of the line.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5994445)